presto-product-tests/src/main/java/com/facebook/presto/tests/hive/AllSimpleTypesTableDefinitions.java

```diff
@@ -86,41 +83,15 @@ private static HiveTableDefinition allHiveSimpleTypesParquetTableDefinition()
                         "   c_bigint             BIGINT," +
                         "   c_float              FLOAT," +
                         "   c_double             DOUBLE," +
+                        "   c_decimal            DECIMAL," +
+                        "   c_decimal_w_params   DECIMAL(10,5)," +
                         "   c_timestamp          TIMESTAMP," +
                         "   c_string             STRING," +
                         "   c_varchar            VARCHAR(10)," +
-                        "   c_char               CHAR(10)," +
-                        "   c_boolean            BOOLEAN" +
-                        ") " +
-                        "STORED AS PARQUET " +
-                        "LOCATION '%LOCATION%'")
-                .setDataSource(dataSource)
-                .build();
-    }
-
-    private static HiveTableDefinition allHiveSimpleTypesKnownToPrestoTextfileTableDefinition()
-    {
-        String tableName = "textfile_all_types_known_to_presto";
-        HiveDataSource dataSource = createResourceDataSource(tableName, "" + System.currentTimeMillis(), "com/facebook/presto/tests/hive/data/all_types_known_to_presto/data.textfile");
-        return HiveTableDefinition.builder(tableName)
-                .setCreateTableDDLTemplate("" +
-                        "CREATE EXTERNAL TABLE %NAME%(" +
-                        "   c_tinyint            TINYINT," +
-                        "   c_smallint           SMALLINT," +
-                        "   c_int                INT," +
-                        "   c_bigint             BIGINT," +
-                        "   c_float              FLOAT," +
-                        "   c_double             DOUBLE," +
-                        "   c_timestamp          TIMESTAMP," +
-                        "   c_date               DATE," +
-                        "   c_string             STRING," +
-                        "   c_varchar            VARCHAR(10)," +
-                        "   c_char               CHAR(10)," +
                         "   c_boolean            BOOLEAN," +
                         "   c_binary             BINARY" +
                         ") " +
-                        "ROW FORMAT DELIMITED FIELDS TERMINATED BY '|' " +
-                        "STORED AS TEXTFILE " +
```

Comment:
This is an unused table definition. I'll make it more clear how I'm removing it.